CODEOWNERS

```diff
@@ -0,0 +1,104 @@
+/cpu/lpc2387/                               @benpicco @maribu
+/cpu/mips*/                                 @kaspar030 @francois-berder
+/cpu/msp430*/                               @kaspar030 @gschorcht
+/cpu/nrf52/radio/nrf802154/                 @bergzand @SemjonKerner @jia200x
+/cpu/sam0_common/                           @benpicco @dylad @keestux
+/cpu/samd21/                                @dylad @biboc
+/cpu/saml1x/                                @dylad
+/cpu/saml21/                                @dylad
+/cpu/stm32*/                                @aabadie @fjmolinas @DipSwitch @vincent-d
+
+/doc/                                       @aabadie @jia200x
+
+/drivers/ad7746/                            @leandrolanzieri
+/drivers/at86rf2xx/                         @daniel-k @Hyungsin @jia200x @smlng @miri64
+/drivers/cc110x/                            @maribu
+/drivers/cc1xxx_common/                     @maribu
+/drivers/ccs811/                            @gschorcht
+/drivers/enc28j60/                          @haukepetersen
+/drivers/dcf77/                             @daexel
+/drivers/dose/                              @jue89
+/drivers/ds18/                              @leandrolanzieri
+/drivers/itg320x/                           @gschorcht
+/drivers/mrf24j40/                          @bergzand
+/drivers/pca9685/                           @gschorcht
+/drivers/sht3x/                             @gschorcht
+/drivers/si70xx/                            @basilfx
+/drivers/sx127x/                            @aabadie @jia200x
+/drivers/ws281x/                            @maribu
+/drivers/xbee/                              @haukepetersen @miri64
+
+/pkg/cryptoauthlib/                         @Einhornhool @PeterKietzmann
+/pkg/lua/                                   @jcarrano
+/pkg/lwip/                                  @miri64
+/pkg/gecko_sdk/                             @basilfx
+/pkg/lora-serialization/                    @leandrolanzieri
+/pkg/micropython/                           @bergzand @kaspar030
+/pkg/openthread/                            @Hyungsin @jia200x
+/pkg/semtech-loramac/                       @aabadie @jia200x
+/pkg/tinydtls/                              @rfuentess @leandrolanzieri
+/pkg/u8g2/                                  @basilfx
+/pkg/ucglib/                                @basilfx
+/pkg/wakaama/                               @leandrolanzieri
+
+/sys/clif/                                  @leandrolanzieri
+/sys/event/                                 @kaspar030
+/sys/evtimer/                               @kaspar030
+/sys/fmt/                                   @kaspar030
+/sys/net/application_layer/gcoap/           @haukepetersen @kb2ma
+/sys/net/application_layer/nanocoap/        @kaspar030 @haukepetersen @kb2ma
+/sys/net/netif/                             @miri64 @jia200x
+/sys/net/gnrc/link_layer/lorawan/           @jia200x
+/sys/net/gnrc/netif/                        @miri64 @jia200x
```

Comment:
You can put me down for `/sys/net/gnrc/network_layer` and `/sys/net/gnrc/transport_layer/udp` as well. Maybe add (even though he is not a maintainer) @brummer-simon for `/sys/net/gnrc/transport_layer/tcp`?

Reply:
Fine by me